SLI_README.md

```diff
@@ -7,6 +7,12 @@ This metric is measuring number of tables which are not backed up since X days f
 
 Metric is implemented as a BigQuery query, which uses data about modifications of tables (collected by Census) and data about existing backups. 
 
+### Number of tables which last backup differs in number of rows or bytes
+
+This metric is measuring number of tables which last available backup does not contain the same data as source.
+
+Metric is implemented as a BigQuery query, which list all tables and joins them with last available backup using [Cloud Datastore export](./SETUP.md#cloud-datastore-export) and then compare the number of bytes and rows between source table and its backup (collected by Census). 
```

Comment:
I would add information that we are verifying tables not changed for at least 3 days

Reply:
Done!